packages/test/test-version-utils/src/versionUtils.ts

```diff
@@ -327,7 +326,7 @@ export function getRequestedRange(baseVersion: string, requested?: number | stri
 export function internalSchema(
 	publicVersion: string,
 	internalVersion: string,
-	requested: number | string,
+	requested: number,
```

Comment:
TS5 no longer allows string/number comparisons (e.g., 'requested <= -2').

Reviewing the code, it looks like the caller handles the case where 'requested' is a string:
```ts
export function getRequestedRange(baseVersion: string, requested?: number | string): string {
	if (requested === undefined || requested === 0) {
		return baseVersion;
	}
	if (typeof requested === "string") {
		return requested;
	}
```